Updating From ARM to Pulumi page (#15662)
pulumi/docs: content/docs/iac/adopting-pulumi/migrating-to-pulumi/from-arm.md

<img src="/logos/tech/azure_arm.png" align="right" class="h-32 px-8 pb-4">

> Tired of slow, rigid ARM templates holding back your Azure infrastructure? Pulumi offers a flexible, code-first approach using C#, Python, Go, or TypeScript.

Reviewer: ```suggestion
Pulumi offers a flexible, code-first alternative to Azure ARM templates using C#, Python, Go, or TypeScript.
```

|-------------------|----------------------|-------------------------------|
| Language          | JSON / YAML          | C# (or Python, TS, Go)        |
| Reuse             | Limited (copy/paste) | Functions, classes, modules   |
| Logic & Loops     | Complex expressions  | if / for / switch             |
| Type Safety       | None                 | Compile-time type checking    |
| Tooling           | Basic validation     | Full IDE & IntelliSense       |
| Testing           | Manual or none       | xUnit, NUnit, etc.            |
| Code Sharing      | Hard                 | NuGet packages & projects     |
| Debugging         | Deploy & troubleshoot| Step-through in IDE           |
| Refactoring       | Tedious manual edits | IDE-assisted refactors        |
| Modularity        | Nested templates     | Real modules & components     |

Pulumi lets you manage Azure infrastructure using your favorite language, with built-in support for CI/CD, reusable components, and testability. For .NET teams especially, Pulumi’s support for C# provides a superior experience over static JSON.

Continue reading to learn how to:

> - [Reference ARM outputs in Pulumi programs](https://www.pulumi.com/docs/iac/adopting-pulumi/migrating-to-pulumi/from-arm/#reference-arm-outputs-in-pulumi-programs)

Reviewer 2: It ( the linter ) wants * rather than - for this list ( and below )

Author: Thanks, Adam! I though bullet points were just -
